.github/workflows/README.md

```diff
@@ -0,0 +1,115 @@
+# Inspektor Gadget CI Workflow
```

Comment:
Could this documentation be moved somewhere in `docs/`?

In this way, documentation websites (such as https://kinvolk.io/docs/inspektor-gadget/latest) could be automatically generated.

We could keep a small `.github/workflows/README.md` redirecting to the file in `docs/`.